openquake/hazardlib/sourceconverter.py

```diff
@@ -960,6 +963,12 @@ def convert_kiteFaultSource(self, node):
 
         msr = valid.SCALEREL[~node.magScaleRel]()
         mfd = self.convert_mfdist(node)
+        try:
+            xstep = valid.float_(~node.floating_x_step)
+            ystep = valid.float_(~node.floating_y_step)
```

Comment:
For these lines I deduce that the XML can contains tags `<floating_x_step>` and `<floating_t_step>`, but where is an example with that? I do not see it. Also, if `<floating_x_step>` and `<floating_t_step>` are in the XML, then it becomes impossible to override such parameters with what is in the job.ini. Do we want that?